clients/src/test/java/org/apache/kafka/common/protocol/ApiKeysTest.java

```diff
@@ -87,4 +88,15 @@ public void testApiScope() {
             "Found some APIs missing scope definition");
     }
 
+    @Test
+    public void testHtmlOnlyHaveStableApi() {
+        String html = ApiKeys.toHtml();
+        for (ApiKeys apiKeys : ApiKeys.clientApis()) {
+            if (apiKeys.messageType.latestVersionUnstable()) {
+                assertFalse(html.contains(apiKeys.name), "Html should not contain unstable api: " + apiKeys.name);
```

Comment:
Could you use `"The_Messages_" + apiKeys.name`? that is more accurate. If the api name is "too" normal, it could be used by other APIs :)